scanner/scanner.go

```diff
@@ -285,8 +285,8 @@ func (s *Scanner) Scan(ctx context.Context, foundCert func(*ct.LogEntry), foundP
 
 	ticker := time.NewTicker(time.Second)
 	startTime := time.Now()
-	fetches := make(chan fetchRange, 1000)
-	jobs := make(chan entryInfo, 100000)
+	fetches := make(chan fetchRange)
+	jobs := make(chan entryInfo)
```

Comment:
How about keeping `jobs` still buffered, but make the size a `ScannerOptions` field?

Reply:
I am fairly convinced buffering belongs in the application, one layer up, not in the generator object.

Most of the work consuming entires from jobs is done by foundCert/foundPrecert so buffering can be implemented there if needed.

But of course, this is your codebase, I won't be offended if you push that change 😄 (GitHub lets you push to my branch) my main goal is to cut the 6GB of fixed buffer.

Reply:
I'd rephrase as "control of buffering belongs in the application, not in the generator object" and then we'd be pretty much in agreement :-)

Reply:
Fair enough :) I can't push right now, so feel free to if you have time, otherwise I'll get to it this week.

Reply:
@daviddrysdale Done